AirBoard/AirBoard/Controllers/AirportTableViewController.swift

```diff
@@ -63,14 +63,11 @@ class AirportTableViewController: UITableViewController {
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cellIdentifier = "AirportCell"
+        let cell = tableView.dequeueReusableCell(withIdentifier: "AirportCell", for: indexPath) as! AirportTableViewCell
```

Comment:
force cast in such cases is not a mistake or bad. Unfortunately for some reason apple can't return exact type of cell so you need to convert it.